semantic_conventions/metrics/process-runtime-jvm-metrics.yaml

```diff
@@ -156,7 +156,7 @@ groups:
       (unlike `system.cpu.utilization`).
       [Reference](https://docs.oracle.com/en/java/javase/17/docs/api/java.management/java/lang/management/OperatingSystemMXBean.html#getSystemLoadAverage()).
     instrument: gauge
-    unit: "1"
+    unit: "{core}"
```

Comment:
I'm not sure `{core}` is correct here. I think system load measures the queue of things waiting (or running) on the cpu, so can be larger than the number of cores.

Reply:
> Is `{core}` correct?

I agree it's not strictly correct. I would argue it's a reasonable simplification that gives the user the right idea, esp. that the value range is not a percentage - a common confussion.

> "unitless" == `1`

IIUC, "unitless" translates to no unit being set, while `1` is a some kind of ratio, e.g. CPU utilization ratio ([spec](https://github.com/open-telemetry/opentelemetry-specification/blob/6a57f094004ededd1f8ec7b9df8e8dfe3983d383/specification/metrics/semantic_conventions/README.md?plain=1#L193-L202)).